### ISSUE TITLE
removeRoleParents is missing

### ISSUE THREAD

**Issue by vonnieda** [OptimalBits/node_acl#87]
There is an addRoleParents, which works great, but there is no way to remove a role parent once you've added it.

**Comment by gitawego** [comment]
yeah, I have the same problem. it would be better to add **removeRoleParents** and **getChildRoles**

**Comment by jurko-gospodnetic** [comment]
Added `removeRoleParents()` in pull request #130.
